composer.json

```diff
@@ -6,28 +6,42 @@
         {
             "name": "Massimiliano Braglia",
             "email": "massimiliano.braglia@fazland.com"
+        },
+        {
+            "name": "Alessandro Chitolina",
+            "email": "alekitto@gmail.com"
         }
     ],
+    "scripts": {
+        "phpstan": "phpstan analyse lib/ -c phpstan.neon --level=8 --no-progress -vvv --memory-limit=2048M",
+        "cscheck": "vendor/bin/phpcs --standard=Solido lib/",
+        "csfix": "vendor/bin/phpcbf --standard=Solido lib/"
+    },
     "require": {
-        "php": ">=7.0",
+        "php": "^7.4|^8.0",
```

Comment:
Since we're updating to modern php requirements, it's preferable to use `||` instead `|`